client/packages/host/src/components/AppDrawer/AppDrawer.tsx

```diff
@@ -173,6 +176,11 @@ export const AppDrawer: React.FC = () => {
     EnvUtils.mapRoute(location.pathname).docs
   }`;
 
+  const handleLogout = () => {
+    navigate(RouteBuilder.create(AppRoute.Login).build());
+    logout();
```

Comment:
Nice yes awesome.

It is looking to me like it would take a bit of work because the authcontext is outside the router context, and I'm thinking maybe it isn't worth making these changes. 

What do you think?

Reply:
Ah yes, okay. In that case, just leave it for now. I suspect we *could* change the context stack so that the Router context is outside Auth, but I don't know for sure what problems that might cause, so let's not open that can of worms just yet :)